docs-core/src/main/resources/db/update/dbupdate-028-0.sql

```diff
@@ -0,0 +1,14 @@
+create memory table T_REVIEW (
+    REV_USERNAME_C varchar(36),
+    REV_RATING_C integer,
+    REV_REVIEWID_C varchar(36) not null,
+    REV_USERID_C varchar(36),
+    REV_DOCID_C varchar(36),
+    REV_REVIEWERID_C varchar(36),
+    REV_CREATEDATE_D date not null,
+    REV_DELETEDATE_D date not null,
```

Comment:
Delete date should not be `not null`, since it is null until the entry is deleted